private/rules/javadoc.bzl

```diff
@@ -20,7 +21,11 @@ def _javadoc_impl(ctx):
 
     classpath = depset(transitive = [dep[JavaInfo].transitive_runtime_jars for dep in ctx.attr.deps])
 
-    generate_javadoc(ctx, ctx.executable._javadoc, sources, classpath, jar_file)
+    # javadoc options have overlap with javac options, but we cannot rely on that derive
```

Comment:
"but we cannot rely on that derive javadoc options from dep[JavaInfo].compilation_info" reads a little odd